src/apps/opening-hours-editor/helper.ts

```diff
@@ -0,0 +1,112 @@
+import dayjs from "dayjs";
+import { EventInput } from "@fullcalendar/core";
+import { EventImpl } from "@fullcalendar/core/internal";
+import { DplOpeningHoursListGET200Item } from "../../core/dpl-cms/model";
+
+const formatDateTimeString = (date: string, time: string): string => {
+  return `${date}T${time}:00`;
```

Comment:
Consider using dayjs for this instead of simple string manipulation.

If we need the data to be in a specific format e.g. ISO8601 they also have helpers for this.